### PR TITLE
STORAUTO-62 support for setting LUN ID on block volumes

### PR DESCRIPTION
**Pull request by causton81**
This depends on FBLOCK-3 being released, but I would like to address any comments in the meantime so it can be ready to go when FBLOCK-3 is released.

### CODE REVIEW

File: SoftLayer/CLI/block/lun.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    """
+
+    block_storage_manager = SoftLayer.BlockStorageManager(env.client)
+
+    res = block_storage_manager.create_or_update_lun_id(volume_id, lun_id)
+
+    if 'value' in res and lun_id == res['value']:
+        click.echo(
+            'Block volume with id %s is reporting LUN ID %s' % (res['volumeId'], res['value']))
+    else:
```
Comment (allmightyspiff) [inline SoftLayer/CLI/block/lun.py]: how does createOrUpdateLunId report errors?
if it throws an exception, this else could be removed
if it returns false this is fine, but if it returns a message, that message should be displayed.

Comment (causton81) [inline SoftLayer/CLI/block/lun.py]: There is a bug that returns null when the underlying property does not exist before the method is called. I am told that is addressed in another issue.

**Comment by allmightyspiff** [comment]
@causton81 looks like my change in #841 is causing a merge conflict with this pull request.

When you get a chance can you update your changes to include mine to resolve this conflict.

Thanks.

**Comment by causton81** [comment]
@allmightyspiff , FBLOCK-3 was deployed and this should be good to merge now.

**Pull request merged**